frame/im-online/src/lib.rs

```diff
@@ -736,16 +831,20 @@ impl<T: Config> Pallet<T> {
 		res
 	}
 
-	fn initialize_keys(keys: &[T::AuthorityId]) {
+	fn initialize_keys(keys: &Vec<T::AuthorityId>) {
```

Comment:
This is a stricter requirement than `&[T::AuthorityId]`, why the change?

Reply:
Can't remember why, but changing it back to a slice works, so I have changed it, thanks